jobbergate-cli/tests/subapps/job_submissions/test_app.py

```diff
@@ -20,8 +20,13 @@ def test_create(
     job_script_id = job_submission_data.id
 
     mocked_render = mocker.patch("jobbergate_cli.subapps.job_submissions.app.render_single_result")
-    patched_create_job_submission = mocker.patch("jobbergate_cli.subapps.job_submissions.app.create_job_submission")
+    patched_create_job_submission = mocker.patch(
+        "jobbergate_cli.subapps.job_submissions.app.create_job_submission",
```

Comment:
does it make any difference adding the comma here?

Reply:
My IDE complained that it was above the max line length (110 characters). Actually, the lint check on the CI should have complained, but I see the Makefile on the CLI does not analyze the directory `tests` as the API does.